conf/layer.conf

```diff
@@ -0,0 +1,12 @@
+# We have a conf and classes directory, append to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have a recipes directory, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes*/*/*.bb ${LAYERDIR}/recipes*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "meta-safeboot"
+BBFILE_PATTERN_meta-safeboot = "^${LAYERDIR}/"
+BBFILE_PRIORITY_meta-safeboot = "6"
+
+LAYERDEPENDS_meta-safeboot = "core openembedded-layer signing-key"
```

Comment:
We needed signing-key layer for sbsigntool but we providde a full sbsigntool recipe here. Do we need something else from this layer?

Which layer provides the tpm2 recipes for our bbappends? Is it in the LAYERDEPENDS var?

Reply:
Fixed